src/Kestrel.Core/Internal/Http2/Http2Frame.GoAway.cs

```diff
@@ -7,7 +7,7 @@ public partial class Http2Frame
     {
         public int GoAwayLastStreamId
         {
-            get => (Payload[0] << 24) | (Payload[1] << 16) | (Payload[2] << 16) | Payload[3];
+            get => (Payload[0] << 24) | (Payload[1] << 16) | (Payload[2] << 8) | Payload[3];
```

Comment:
This is the second time I've seen this bug. I did a search for all instances of `<< 16.+<< 16` in our code base in a single line, and it seems like there's a another instance of this bug in Http2Frame.RstStream.cs in RstStreamErrorCode. Might was well fix it there too.